### ISSUE TITLE
New client: function of "Restore from Backup" is unclear (or doesn't work?)

### ISSUE THREAD

**Issue by atomrab** [periodo/periodo-client#278]
I'm looking at the new client on a Chrome browser (Version 83.0.4103.116, 64 bit) on a Macbook Pro. I assume that the "Restore from backup" button is meant to load an existing file into the client, in the way that one used to be able to share local files. I tried it with an export in .json format from the old client, but nothing happened when I clicked on the "Open" button in the file-finder window, except that the "Choose backup file" developed a blue border.

Then I thought that this might want a file formatted by the new client -- but having made a new local IDB, I can't see a way to download it or back it up. So what exactly am I supposed to load using the "Restore from Backup" function?

**Comment by atomrab** [comment]
Update: I now see that I can create a backup data source if I click on "edit" for the local collection.

**Comment by atomrab** [comment]
And I have successfully restored it from backup. Ok, never mind -- this works; it just needs documentation.

**Comment by ptgolden** [comment]
Yeah-- something dropped in without documentation. I'll make it work with just a plain dataset as well.
